remoting/consul/agent.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
What the consultAgent used for ?

Reply:
If used for unit test, it's better to move to xxx_test file.

Reply:
This `consulAgent` is used for test, and it is used for test and imported by test. I rename `agent.go` to `test_agent.go`.